sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -65,4 +95,25 @@ class ProjectOverviewView extends VerticalLayout{
         def dataProvider = new ListDataProvider(viewModel.projectOverviews)
         projectGrid.setDataProvider(dataProvider)
     }
+
+    private TextArea setupManifestContent() {
+        TextArea textArea = new TextArea("Download Manifest")
+        textArea.setReadOnly(true)
+        textArea.setVisible(false)
+        viewModel.addPropertyChangeListener("generatedManifest", {
+            textArea.setValue(Optional.ofNullable(it.newValue).orElse("") as String)
+        })
+        return textArea
+    }
+
+    private AbstractComponent setupProjectSpecificButtons() {
+
+        MenuBar buttonBar = new MenuBar()
+        buttonBar.addItem("Download Project", {
```

Comment:
You do not download the project but the manifest file for the samples. Maybe just call it Download Data or Download Manifest. (You can download data when the project is not completed too)

Reply:
Called it "Get Available Data", also good?